examples/tut_matrix-multiply.cpp

```diff
@@ -837,11 +837,11 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
           RAJA::statement::Tile<0, RAJA::statement::tile_fixed<CUDA_BLOCK_SIZE>, RAJA::cuda_block_x_loop,
             RAJA::statement::For<1, RAJA::cuda_thread_y_loop, // row
               RAJA::statement::For<0, RAJA::cuda_thread_x_loop, // col
-                RAJA::statement::Lambda<0>,   // dot = 0.0
+                RAJA::statement::Lambda<0, RAJA::statement::Params<0>>,   // dot = 0.0
                 RAJA::statement::For<2, RAJA::seq_exec,
                     RAJA::statement::Lambda<1> // dot += ...
                 >,
-                RAJA::statement::Lambda<2>   // set C = ...
+                RAJA::statement::Lambda<2, RAJA::statement::Segs<0, 1>, RAJA::statement::Params<0>>   // set C = ...
```

Comment:
One thing I've been wanting to do is shorten the namespace for ``Segs`` and ``Params``. Could this just be under the ``RAJA`` namespace? What do you guys think?

Reply:
@artv3 and @rhornung67 , i agree.  I really intended only the actual statements to be in that namespace (For, Tile, Lambda, etc).  How about I move everything else back into RAJA::?

Reply:
❤️

Reply:
It wouldn't really hurt anything I suppose, but I'm not sure it's all that big a win either. As it is now one can always `using` or `using namespace` them in from where they are to cut all of that out.